_posts/2019-07-29-PELUX-4.0.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "PELUX 4.0 Released!"
+date:   2019-07-29 15:00:00 +0100
+categories: yocto release
+author: Oleksandr Kravchuk
+---
+
+PELUX 4.0 is here!
+
+This major release among the other contains the following changes:
+- Adaptation to Yocto "Thud" 2.6
+- Integration with Qt AS 5.13
+- Experimental [Mopidy playback plugin](https://github.com/Pelagicore/qtivi-mopidy-plugin) for Qt IVI
+- [Connectivity Manager](https://github.com/Pelagicore/connectivity-manager) for managing network connectivity
```

Comment:
Rephrase to, Connect to Wi-Fi using Connectivity Manager

Reply:
hm.... @martin-ejdestig any comments?